### PR TITLE
Update: Trino, Presto Logic and Doc

### PR DESCRIPTION
**Pull request by MilanBariya**
### Describe your changes :

I worked on updating Trino, Presto doc and logic.

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Review by ulixius9 — DISMISSED** [review]

**Review by pmbrull — COMMENTED** [review]
Thanks for validating these!

Just a qq, any user management we should take into account for the docs?

**Comment by MilanBariya** [comment]
> Thanks for validating these!
> 
> Just a qq, any user management we should take into account for the docs?

I will check that out.

File: openmetadata-docs/content/connectors/database/trino/index.md [hunk 1]
```diff
@@ -43,6 +43,17 @@ To deploy OpenMetadata, check the <a href="/deployment">Deployment</a> guides.
 To run the Ingestion via the UI you'll need to use the OpenMetadata Ingestion Container, which comes shipped with
 custom Airflow plugins to handle the workflow deployment.
 
+<Note>
+
+To Inesget metadata from the Trino User Must have select privileges to this tables.
```
Comment (ulixius9) [inline openmetadata-docs/content/connectors/database/trino/index.md:48]:
```suggestion
To ingest metadata from the Trino source, the user must have select privileges for the following tables.
```

**Review by NiharDoshi99 — APPROVED** [review]

**Review by ulixius9 — APPROVED** [review]

**Comment by MilanBariya** [comment]
> Thanks for validating these!
> 
> Just a qq, any user management we should take into account for the docs?

For Permissions, I will create a new PR, As of now merging this.

**Pull request merged**